Formula/gst-plugins-bad.rb

```diff
@@ -38,14 +38,14 @@ class GstPluginsBad < Formula
   depends_on "rtmpdump" => :optional
   depends_on "schroedinger" => :optional
   depends_on "sound-touch" => :optional
-  depends_on "srtp" => :optional
+  depends_on "srtp@1.5" => :optional
```

Comment:
If it does not compile with current SRTP, I recommend we simply remove this option, which is used in 0.6% of installs